example/App.vue

```diff
@@ -6,19 +6,25 @@
     <p>helloMsg: {{helloMsg}}</p>
     <p>computed msg: {{computedMsg}}</p>
     <button @click="greet">Greet</button>
+    <hello ref="helloComponent"></hello>
   </div>
 </template>
 
 <script lang="ts">
 import Vue from 'vue'
 import Component from '../lib/index'
+import Hello from './Hello.vue';
 
 @Component({
   props: {
     propMessage: String
+  },
+  components: {
+    Hello
   }
 })
 export default class App extends Vue {
+  // props are had to declare again
```

Comment:
`props had to be declared again` ?

Reply:
How about `You have to declare props again.` ?

Reply:
Though the `prop` option in  `@Component` is not a declaration, I think it is necessary to tell the starter why we have to declare `propMessage` again.

Reply:
Doesn't it grammatically invalid that `props are had to`? I think we usually say `props have to be` in that case (sorry, my previous comment should be "have" instead of "had")

Reply:
Yes, my fault. 😆 I'll fix it now.
`props have to be declared for typescript`
Is it nicer or not ?